test/instances/storage.lua

```diff
@@ -108,6 +108,18 @@ local function bucket_gc_continue()
     vshard.storage.garbage_collector_wakeup()
```

Comment:
> This should be fixed separately. Most likely by addition of sanity
checks to the _bucket:on_replace trigger to raise an error when
there is a threat of data loss or corruption.

I suggest to file an issue just in order not to forget

Reply:
It is the same issue. I didn't close 173th by this PR. There will be part 4.

Reply:
Ok